classes/label.rb

```diff
@@ -0,0 +1,14 @@
+class Label
+  def initialize(title, color, id = Random.rand(1..1000))
+    @id = id
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
- [ ] Please know that you need to specify the correct 1-to-many-relationship here, this is an example of how it should be: 
```ruby
    def add_item(item)
      @items.push(item)
      item.genre = self
    end
```